File: docs/src/pages/scrolling/snippets/scroll-target/both-axes.js
SpreadGrid(div, {
    // collapse: true
    data: data,
    formatting: [
        { row: { id: 25 }, style: { background: '#cce5ff' } },
        { column: { id: 'c25' }, style: { background: '#cce5ff' } },
    ],
    pinnedTop: 1,
    pinnedLeft: 1,
    // collapse: false
    verticalScrollTarget: { index: 25 },
    horizontalScrollTarget: { index: 25 },
    verticalScrollSpeed: 'smooth',
    horizontalScrollSpeed: 'smooth',
});

File: docs/src/pages/scrolling/snippets/scroll-target/both-axes.py
app.layout = DashSpreadGrid(  # type: ignore
    # collapse: true
    data=data,
    formatting=[
        {"row": {"id": 25}, "style": {"background": "#cce5ff"}},
        {"column": {"id": "c25"}, "style": {"background": "#cce5ff"}},
    ],
    pinned_top=1,
    pinned_left=1,
    # collapse: false
    vertical_scroll_target={"index": 25},
    horizontal_scroll_target={"index": 25},
    vertical_scroll_speed="smooth",
    horizontal_scroll_speed="smooth",
)
